Tasks/AzurePowerShellV4/PreJobExecutionAzurePowerShell.ps1

```diff
@@ -0,0 +1,50 @@
+. "$PSScriptRoot/PreJobExecutionUtility.ps1"
+
+$targetAzurePs = Get-VstsInput -Name TargetAzurePs
```

Comment:
Why we are reading this inputs here?

Reply:
yes we do not need to read them here, unless you want to read them and set the psmodulepath variable here. Otherwise these changes are not needed.

Reply:
done